google-cloud-logging/lib/google/cloud/logging/rails.rb

```diff
@@ -82,12 +89,13 @@ class Railtie < ::Rails::Railtie
         ##
         # Determine whether to use Stackdriver Logging or not.
         #
-        # Returns true if valid GCP project_id is provided and underneath API is
-        # able to authenticate. Also either Rails needs to be in "production"
-        # environment or config.stackdriver.use_logging is explicitly true.
+        # Returns `true` if a valid GCP `project_id` is provided and the API is
+        # able to authenticate. Also, either Rails needs to be run in the
+        # `production` environment or `config.stackdriver.use_logging` is
+        # explicitly set to `true`.
```

Comment:
It's a bit awkward. How about:

```
Returns `true` if Stackdriver Logging is enabled for this application---that is, if all of the following are true:
* A valid GCP `project_id` is available, either because the application is hosted on Google Cloud or because it is set in the configuration.
* The API is able to authenticate, again either because the application is hosted on Google Cloud or because an appropriate keyfile is provided in the configuration.
* Either the Rails environment is set to `production` or the `config.google_cloud.use_logging` configuration is explicitly set to `true`.
```

Also, in either case, s/stackdriver/google_cloud/